_components/form-controls/10-file-input.md

```diff
@@ -0,0 +1,54 @@
+---
+title: File input
+parent: Form controls
+order: 10
+lead: File inputs allow users to attach a file
+---
+
+{% include code/preview.html component="file-input" %}
+{% include code/accordion.html component="file-input" %}
+
+<div class="usa-accordion usa-accordion--bordered site-accordion-docs">
+  <button class="usa-button-unstyled usa-accordion__button"
+      aria-expanded="true" aria-controls="file-input-docs">
+    Guidance
+  </button>
+  <div id="file-input-docs" aria-hidden="false" class="usa-accordion__content site-component-usage">
+
+    <h4>When to use the file input</h4>
+    <ul class="usa-content-list">
+      <li><strong>Documents are required.</strong> You should only ask users to upload files when it is critical to service delivery.</li>
```

Comment:
You use the word "upload" here but later mention that this component doesn't upload files. Maybe use the word, attach?

Reply:
Good point - better to be precise. I've reworded this "provide"